cfme/cloud/security_group.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-
+import attr
+
+from cfme.base.ui import BaseLoggedInPage
+from cfme.exceptions import ItemNotFound, SecurityGroupsNotFound
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigate_to, navigator
+from cfme.utils.wait import wait_for
+from navmazing import NavigateToAttribute, NavigateToSibling
```

Comment:
This is a pip-installed import, along with widgetastic+patternfly imports below. They should be separated from local imports and above them (below and separate from `attr` import)